### PR TITLE
added jazz-hands package

### PR DESCRIPTION
**Pull request by athulvis**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by dopplershift — CHANGES_REQUESTED** [review]
The recipe's in pretty good shape, just a few minor things. @tzdwi @theand9 @gjgilbert @astroluv can you confirm that you are each willing to be a maintainer for this recipe?

**Comment by dopplershift** [comment]
@athulvis You can just commit the necessary changes to your local branch and push them up, no need to make a new PR.

**Comment by tzdwi** [comment]
> The recipe's in pretty good shape, just a few minor things. @tzdwi @theand9 @gjgilbert @astroluv can you confirm that you are each willing to be a maintainer for this recipe?

Yep, I can be a maintainer!

**Comment by theand9** [comment]
> The recipe's in pretty good shape, just a few minor things. @tzdwi @theand9 @gjgilbert @astroluv can you confirm that you are each willing to be a maintainer for this recipe?

Yes, I would love to be a maintainer! Thanks.

**Comment by gjgilbert** [comment]
> The recipe's in pretty good shape, just a few minor things. @tzdwi @theand9 @gjgilbert @astroluv can you confirm that you are each willing to be a maintainer for this recipe?

Yes, I'm happy to

File: recipes/jazz-hands/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jazz-hands-{{ version }}.tar.gz
+  sha256: 86e1e57962f7f12942e8439af8566a941153c2e214c5e0992e24698f536b71cd
+
+build:
+  number: 0
+  entry_points:
+    - astropy-package-template-example = packagename.example_mod:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
```
[the remark this answers is not in the document]
Comment (athulvis) [reply recipes/jazz-hands/meta.yaml:18]: @dopplershift Adding `noarch: python` shows this lint 
```
. has some lint:
  noarch: python recipes are required to have a lower bound on the python version.
 Typically this means putting `python >=3.6` in **both** `host` and `run` but you should 
check upstream for the package's Python compatibility.
```
How can I avoid this?

[the remark this answers is not in the document]
Comment (athulvis) [reply recipes/jazz-hands/meta.yaml:18]: Sorry. Fixed it. Added version for python in requirements.

**Comment by athulvis** [comment]
@dopplershift I have updated the recipe with the suggested changes. Thank you

**Comment by athulvis** [comment]
@dopplershift  I have updated the suggested changes. Please have a look. Sorry for my PR deletions.

**Review by dopplershift — APPROVED** [review]
Looks good. Just need confirmation from @astroluv that they are willing to be a maintainer.

**Comment by athulvis** [comment]
@dopplershift Thank you. I have a doubt. Can I add other maintainers later , if needed? Can I invoke new PR for that if the current one is appproved?

**Comment by dopplershift** [comment]
Yes, you can always add new maintainers once the feedstock has been created. There's even a [bot command](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user) to make the PR for you without triggering a build.

**Comment by astroluv** [comment]
Yes, I can be a maintainer!

Sorry for the delay everyone!

Sincerely,

Teresa

**Comment by athulvis** [comment]
Thank you!!!

**Pull request merged**